.gitignore

```diff
@@ -11,4 +11,11 @@ coverage
 .DS_Store
 airseeker.json
 secrets.env
-.eslintcache
\ No newline at end of file
+.eslintcache
+
+# Hardhat files
```

Comment:
These were generated by `hardhat --init`. I am not sure if all of them are needed, but it's better to keep them all.